src/tests/Microsoft.PowerFx.Interpreter.Tests/Scenarios/PrimitiveWrapperAsUnknownObject.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Tests
+{
+    // Wrap a .net object as an UntypedObject.
+    // This will lazily marshal through the object as it's accessed.
+    [DebuggerDisplay("{_source}")]
+    public class PrimitiveWrapperAsUnknownObject : IUntypedObject
```

Comment:
>PrimitiveWrapperAsUnknownObject [](http://example.com/codeflow?start=17&length=31)

This is very useful for other authors. IT might be a good candidate to be in the product.